xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/RssMacro.java

```diff
@@ -128,21 +130,16 @@ public boolean supportsInlineMode()
         SyndFeed feed = this.romeFeedFactory.createFeed(parameters);
 
         if (parameters.isDecoration()) {
-            BoxMacroParameters boxParameters = new BoxMacroParameters();
             boolean hasImage = parameters.isImage() && (feed.getImage() != null);
-            boxParameters.setCssClass(FEED_CLASS_VALUE);
+            parameters.setCssClass(FEED_CLASS_VALUE);
 
-            if (StringUtils.isNotEmpty(parameters.getWidth())) {
-                boxParameters.setWidth(parameters.getWidth());
-            }
-
-            boxParameters.setBlockTitle(generateBoxTitle("rsschanneltitle", feed));
+            parameters.setBlockTitle(generateBoxTitle("rsschanneltitle", feed));
```

Comment:
Instead of modifying `parameters` I would introduce something like a `AbstractBoxMacro#getTitle(P parameters, String content, MacroTransformationContext context)` and override it in RssMacro.

Reply:
So you suggest a new refactoring in AbstractBoxMacro then? We're a little short in time, no?

Reply:
I suggest to move one like in a dedicated protected method, not exactly a big refactoring.

Reply:
Sure. But I need to create the dedicated PR in xwiki-rendering, to be reviewed and merged and then this one to be fixed. And the release needs to wait in the meantime...